components/edit-collective/sections/Policies.js

```diff
@@ -276,6 +284,14 @@ const Policies = ({ collective, showOnlyExpensePolicy }) => {
             />
           </Container>
         )}
+        <Container mt={4}>
+          <StyledCheckbox
+            name="allow-expense-submission"
+            label={formatMessage(messages['expensePolicy.allowExpense'])}
+            onChange={() => (formik.values.disablePublicExpenseSubmission = !formik.values.disablePublicExpenseSubmission)}
```

Comment:
We should avoid mutating formik's state directly, the lib is not notified of the change otherwise and this can later cause trouble with validations (and some React warnings, depending on how it's implemented). We can use `setFieldValue`:

```suggestion
            onChange={() => setFieldValue('disablePublicExpenseSubmission', !formik.values.disablePublicExpenseSubmission)}
```

Reply:
Done. 👍🏽

Reply:
Sorry it actually was `formik.setFieldValue`